spec/features/issues_spec.rb

```diff
@@ -16,10 +16,10 @@
     context 'with an Issue library' do
       let(:issuelib) { current_project.issue_library }
 
-      describe 'index page' do
+      describe 'index page', js: true do
         it 'presents a link to add new issue' do
           visit project_issues_path(current_project)
-          expect(page).to have_xpath("//a[@href='#{new_project_issue_path(current_project)}']")
+          expect(page).to have_xpath("//a[@href='#{new_project_issue_path(current_project)}']", visible: false)
```

Comment:
Why is the link hidden?

Reply:
I'm not sure why this started to fail now because it's always been hidden. The user has to click the + sign first then a dropdown is revealed with links to a new issue (blank, default, templates)

Reply:
Ah I think it's due to adding js to the spec. In that case, this also needs to test the clicking of the + sign instead of artificially clicking a hidden link.